backend/external_canisters/sns_root/api/src/queries/get_sns_canisters_summary.rs

```diff
@@ -10,6 +10,7 @@ pub struct Response {
     pub governance: Option<CanisterSummary>,
     pub ledger: Option<CanisterSummary>,
     pub swap: Option<CanisterSummary>,
+    pub dapps: Vec<CanisterSummary>,
```

Comment:
Is there a better name I wonder?

Reply:
That's the name coming from the SNS root canister